frontend/packages/dev-console/src/components/health-checks/health-check-probe-utils.ts

```diff
@@ -1,5 +1,13 @@
 import { HealthChecksProbeType, RequestType, HealthCheckProbe } from './health-checks-types';
 
+export const fetchContainerPorts = (ports) => {
```

Comment:
Fetch actually makes it feel like your doing an fetch API request.
```suggestion
export const getContainerPorts = (ports) => {
```